### DISCUSSION TITLE
Waiting on Ip error

### DISCUSSION THREAD

**Discussion by strategybase** [vmware/packer-examples-for-vsphere#621] [Help]
I keep getting a waiting on ip error when i run the build.sh to create the 20-04-lts template. It times out. Are there config changes that i should be making to go past this error?

**Comment by tenthirtyam** [comment]
- Is this the only build that times out?
- Are you using DHCP?
- Are you using the HTTP or disk based datasource?
- Can the system being built reach back to the packer host? 
- Are they in the same L2?
